pants

```diff
@@ -356,11 +357,11 @@ function bootstrap_pants {
       (
         scrub_PEX_env_vars
         # shellcheck disable=SC2086
-        "${python}" "${virtualenv_path}" --no-download "${staging_dir}/install" && \
+        "${python}" "${virtualenv_path}" --quiet --no-download "${staging_dir}/install" && \
```

Comment:
Does `--quiet` still render all of the output on failure? If not, it might be better to pipe it to a temporary log, and cat the log on failure of this block.

Reply:
errors still show up.

<img width="892" alt="Screen Shot 2022-02-23 at 7 38 29 PM" src="https://user-images.githubusercontent.com/1268088/155434802-c5b03179-437b-4f42-9560-4a90f6545dd3.png">